### PR TITLE
DC-36 get data from aem

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description
"Sample" code to get data from the AEM API (if available) or from a local file 

List of proposed changes:
- Sample code on index,js that gets the data from for the page
- API code file.  

### What to test for/How to test
- N/A for now

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const builddate = process.env.NEXT_PUBLIC_BUILD_DATE
     process.env.NEXT_PUBLIC_BUILD_DATE.substr(6, 2)
   : 'DATE-NA'
 
+// AEM base end point
+var contentURL = process.env.NEXT_CONTENT_API
```
Comment (nibivi77) [inline next.config.js:11]: If possible, can we change `var` to a `const`?

**Review by nibivi77 — APPROVED** [review]

**Comment by nibivi77** [comment]
Looks good to me Marco!

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_BUILD_DATE = 20210917
```
Comment (DaceyTom2) [inline .env.example:1]: .env needs to be added to git ignore

File: pages/api/getData.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return { localData: [] }
+}
+
+//
+// Fetch ALL topics from API when available
+//
+
+export async function getTopics(language) {
+  const res = await fetch(
+    `${process.env.NEXT_CONTENT_API}${language}/lj-new-child.json?date=${process.env.NEXT_PUBLIC_BUILD_DATE}`
```
Comment (DaceyTom2) [inline pages/api/getData.js:23]: If lj-new-child was a parameter would we be able to make something reusable? Perhaps there is a framework for AEM requests?

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: .env.example (hunk 2 above)
Comment (MarcoGoC) [reply .env.example:1]: next uses .env.local

File: pages/api/getData.js (hunk 3 above)
Comment (MarcoGoC) [reply pages/api/getData.js:23]: we could but it depends on how we implement the AEM structure

File: .env.example (hunk 2 above)
Comment (DaceyTom2) [reply .env.example:1]: My apologies, I didn't realise this was an example

File: pages/api/getData.js (hunk 3 above)
Comment (DaceyTom2) [reply pages/api/getData.js:23]: Sounds good

**Review by DaceyTom2 — APPROVED** [review]
Looks good!

**Pull request merged**